src/main/kotlin/com/autonomousapps/internal/models.kt

```diff
@@ -301,10 +301,10 @@ data class Method internal constructor(val types: Set<String>) {
 }
 
 data class AnnotationProcessor(
-  val dependency: Dependency,
+  override val dependency: Dependency,
```

Comment:
@autonomousapps is it the right way to do it ? `AnnotationProcessor` now implements `HasDependency`.

Reply:
Yes, assuming you need to use this somewhere that requires `HasDependency`, this is the right way to do it.

I don't even mind if this stays when you remove the usage that required it (facadeFilter) -- assuming you agree with me there.